### PR TITLE
enable fastScroll for RecyclerView

### PR DESCRIPTION
**Pull request by adakoda**
## Overview (Required)
- This patch enable fastScroll for RecyclerView
- User can scroll view by fastScroller thumb and recognize current position in RecyclerView

## Links
- [Support Library Revision 26.0.0 Release](https://developer.android.com/topic/libraries/support-library/revisions.html#26-0-0)

> New APIs
> New fastScrollEnabled boolean flag for RecyclerView. If enabled, 
> fastScrollHorizontalThumbDrawable, fastScrollHorizontalTrackDrawable, 
> fastScrollVerticalThumbDrawable, and fastScrollVerticalTrackDrawable must be set.

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
Can you merge with a master branch? 
I’m worried about conflict with this
https://github.com/DroidKaigi/conference-app-2018/pull/318

File: app/src/main/res/layout/fragment_about_this_app.xml [hunk 1]
```diff
@@ -15,6 +15,11 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:orientation="vertical"
+            app:fastScrollEnabled="true"
+            app:fastScrollHorizontalThumbDrawable="@drawable/state_fast_scroll_thumb"
+            app:fastScrollHorizontalTrackDrawable="@drawable/state_fast_scroll_track"
+            app:fastScrollVerticalThumbDrawable="@drawable/state_fast_scroll_thumb"
+            app:fastScrollVerticalTrackDrawable="@drawable/state_fast_scroll_track"
```
Comment (konifar) [inline app/src/main/res/layout/fragment_about_this_app.xml]: 📝

**Comment by konifar** [comment]
Thanks!
How about creating `@style/BaseRecyclerView` and apply it in layout?

**Comment by adakoda** [comment]
Thank you for your comment.
I had been added a style for RecyclerView and apply it in target layout.

Regarding #318,
Even If you had been merge my PR, you can revert this changes for specific RecyclerView such as "Contributors" view.
I think that this patch will be good work for "About this app", "Sponsor" view and so on.

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -40,4 +40,11 @@
         <item name="android:layout_width">48dp</item>
         <item name="android:layout_height">48dp</item>
     </style>
+
+    <style name="BaseRecyclerView">
+        <item name="fastScrollHorizontalThumbDrawable">@drawable/state_fast_scroll_thumb</item>
```
Comment (konifar) [inline app/src/main/res/values/styles.xml]: Can you add `fastScrollEnabled` too?

**Comment by adakoda** [comment]
I had been checked both #318(sticky label) and this #335(fastScroller) features in SearchSpeakersFragment. 
The result is **OK**, User can use both features !
There is **nothing conflict**.

checked

**Comment by takahirom** [comment]
Looks good! We can search session fast👍

**Pull request merged**